### PR TITLE
Add SIGN_KEY environment variable to choose GPG signing key

### LINKED ISSUE #17: Add ability to choose GPG signing key
Hi,

I have multiple GPG keys on my machine and I want to be able to choose which one AppImageKit uses for signing. There is a `--sign-key` flag for [appimagetool](https://github.com/AppImage/AppImageKit#appimagetool-usage) for this purpose, however when using `linuxdeploy`, this flag is not exposed. It would be great if we could set it using `linuxdeploy`.

### PR DESCRIPTION
**Pull request by ba32107**
Fixes #17

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -113,6 +113,12 @@ int main(const int argc, const char* const* const argv) {
 
     if (getenv("SIGN") != nullptr) {
         args.push_back(strdup("-s"));
+
+        const char *signingKey;
```
Comment (TheAssassin) [inline src/main.cpp:117]: Code style:

```suggestion
        const char* signingKey;
```

**Review by TheAssassin — COMMENTED** [review]
Thanks, looks good to me except for one minor code style issue. I've suggested a fix.

**Comment by ba32107** [comment]
Thanks! When is the new version going to be bundled in with `linuxdeploy`? I know the docs says the plugins are likely to be outdated. I'm using it automatically from CI, so ideally I wouldn't want to separately download the plugin as well, would prefer to have just one file to work with.

**Comment by TheAssassin** [comment]
You can always download this plugin next to linuxdeploy and make it executable, it will take precedence over the bundled one.

I'll rebuild linuxdeploy on Travis to update the binaries.

**Comment by ba32107** [comment]
Thanks, that would be helpful!

**Comment by TheAssassin** [comment]
The binaries have been built and published, by the way.

**Pull request merged**